src/dateutils.ts

```diff
@@ -78,11 +78,15 @@ export function isToday(date?: XDate | string) {
 }
 
 export function isGTE(a: XDate, b: XDate) {
-  return b.diffDays(a) > -1;
```

Comment:
What was returned before? 
Did it throw an error or return some value

Reply:
It throw en error: Cannot read property 'getTime' of undefined